src/test/integration/ac-auth-mocking.test.ts

```diff
@@ -23,20 +19,24 @@ describe('Mocking AcAuth', () => {
 
     }
 
-    const app = new Application();
-    app.container.rebind(AcAuthProvider).toConstantValue({
-        async provide() {
-            return new AcAuth({
-                jwtContext: {
-                    organisation_id: 'foo',
-                    service_account_id: 'service-account-worker',
-                    service_account_name: 'Bot',
+    class App extends Application {
+        override defineHttpRequestScope(mesh: Mesh) {
```

Comment:
Is it possible to use `super` instead?
```ts
class App extends Application {
    createHttpRequestScope() {
        const mest = super.createHttpRequestScope();
        mesh.service(MyRouter);
        mesh.constant(AcAuthProvider, { ..... });
    }
}
```
Would be nice to avoid adding questionable `defineHttpRequestScope(_mesh: Mesh) {}` to the `App`.

Reply:
Yeah, exactly what I wanted to discuss 👍 

Basically this way you save a couple lines of code when you define your request scoped components, e.g.

```ts
override defineHttpRequestScope() {
    // useful code
}

// vs

override createHttpRequestScope() {
    const mesh = super.createHttpRequestScope();
    // useful code
    return mesh;
}
```

Note: I still would like to keep `createHttpRequestScope` because that would allow extracting the scopes into different files when you really [want/need](https://github.com/ubio/ada/blob/main/backend/src/main/GlobalScope.ts#L44) that — so maybe that `super.` stuff isn't that bad actually.